doctests/home-json.js

```diff
@@ -0,0 +1,149 @@
+// EXAMPLE: js_home_json
+// REMOVE_START
+import assert from 'assert';
+// REMOVE_END
+// STEP_START import
+import {
+    createClient,
+    SchemaFieldTypes,
+    AggregateGroupByReducers,
+    AggregateSteps,
+} from 'redis';
+// STEP_END
+
+// STEP_START connect
+const client = await createClient();
+await client.connect();
+// STEP_END
+// REMOVE_START
+try {
+    await client.ft.dropIndex('idx:users', { DD: true });
+} catch{}
+
+await client.del('user:1', 'user:2', 'user:3');
+// REMOVE_END
+
+// STEP_START create_data
+var user1 = {
+    name: "Paul John",
```

Comment:
use `'` instead of `"`
use `const`/`let` instead of `var`
add missing `;`

Reply:
Found/replaced these throughout the file now.